src/actions/searchActions.js

```diff
@@ -100,7 +100,7 @@ const getSearchParams = (state) => {
     query_fields: state.search.query_fields,
     sort_by_published_date_order: state.search.sortByPublishedDate,
     partial_match: state.search.partialMatch,
-    mod_abbreviation: state.isLogged.oktaMod
+    mod_abbreviation: state.isLogged.testerMod !== 'No' ? state.isLogged.testerMod : state.isLogged.oktaMod
```

Comment:
Does this handle the case where there is no okta login and no tester value ?  (I don't know if mod_abbreviation is required for this)

Reply:
Good question! In this special case, mod_abbreviation is null, the main PDF identifying code will return nothing so no main PDF icon will show up on the search result page.

Currently, the PDF icon for a paper with a main PDF uploaded by SGD, does not appear on the search results page when viewed by a non-SGD curator, even if they are logged in as an SGD tester. With this update, we will set mod_abbreviation = 'SGD' when a non-SGD curator is logged in as an SGD tester, making sure the main PDF icon is visible for a tester as well.

Reply:
Oh... cool cool, thanks !